### DISCUSSION TITLE
Link to my (public) repository returns 404 page and others cannot find it (or my profile)

### DISCUSSION THREAD

**Discussion by Unclearogre** [community/community#73275] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I developed a module for MagicMirror and published it on GitHub -- at least I thought I did.  I posted about it in the MagicMirror forum and on Reddit and the feedback I got was that the link was broken.  

I opened a private browser window (so as to be logged out) and, sure enough, I got the same results.  I then started doing searches (on the GitHub site) to try and find either my repository or my profile.  Neither returned any results.  I thought maybe one had to be signed in (didn't make much sense because I had been able to view other users & repos before I created an account) so I made a second account and signed in but still no luck.  

My repository is at: https://github.com/Unclearogre/MMM-MailMessage

Any thoughts?

**Comment by Unclearogre** [comment, marked as answer]
Solved -- It appears I was shadowbanned for some reason.  It has been fixed now.

**Reply by Duhell** [reply]
How did you fixed it?

**Reply by ARTIFICI4L** [reply]
Can you help me aswell having the same issue

**Reply by CheapFuck** [reply]
same here, how did you fix?

**Reply by idfan92** [reply]
How did you fix it? The issues I submitted also don't appear when I try to search for them on an incognito browser window or on a device I'm not logged in.

**Reply by redtoex** [reply]
same with me
